### PR TITLE
Issue619 - Use Schema's create function & fix tests

### LINKED ISSUE #619: Refactor the post method in the Tickets resource
Refactor the post method in the Tickets resource to use the new create method provided by the base resource class.

### PR DESCRIPTION
**Pull request by Chris-Boe**
## Description
Fixes some test data and uses the schema's create functionality in the ticket POST method

### What issue is this solving?

Closes codeforpdx/dwellingly-app/issues/619

### CODE REVIEW

File: resources/tickets.py [hunk 1]
```diff
@@ -70,11 +71,8 @@ def get(self):
 
     @pm_level_required
     def post(self):
-        data = Tickets.parser.parse_args()
-        ticket = TicketModel(**data)
-
-        ticket.save_to_db()
-        return ticket.json(), 201
+        TicketModel.create(schema=TicketSchema, payload=request.json)
+        return {"message": "Ticket successfully created"}, 201
```
Comment (Chris-Boe) [inline resources/tickets.py:39]: We are not using the returned JSON of the created ticket at all in the frontend (PR: https://github.com/codeforpdx/dwellingly-app/pull/611) so I changed this to just return a success message

**Review by NickSchimek — CHANGES_REQUESTED** [review]
Thanks for the PR. We have someone fixing the seed file in a different issue. Rather than modifying the seed file. I would prefer to disconnect the test from the seed file by using factories.

File: tests/integration/test_tickets.py [hunk 2]
```diff
@@ -58,26 +58,13 @@ def test_tickets_POST(client, auth_headers):
...
-    assert response.json["assignedUserID"] == 4
-    assert response.json["sender"] == "user1 tester"
-    assert response.json["assigned"] == "Mr. Sir"
-    assert response.json["status"] == TicketStatus.New
-    assert response.json["urgency"] == "low"
-
-    # verify jwt only
-    response = client.post(endpoint, json=newTicket, headers=auth_headers["admin"])
-    assert is_valid(response, 201)
+    assert response.json == {"message": "Ticket successfully created"}
```
Comment (NickSchimek) [inline tests/integration/test_tickets.py:67]: There is still some work that needs to be done with this test, but I will go ahead and merge for now. This test isn't actually testing anything useful. It's just testing that we return json, but not actually testing that we called the create method. See the discussion here: https://github.com/codeforpdx/dwellinglybackend/pull/275#discussion_r637637833

**Review by NickSchimek — APPROVED** [review]
Thanks!

**Pull request merged**